homeassistant/components/tts/__init__.py

```diff
@@ -380,20 +386,20 @@ class Provider(object):
     hass = None
     language = None
 
-    def get_tts_audio(self, message):
+    def get_tts_audio(self, message, language):
```

Comment:
set to None -> `get_tts_audio(self, message, language=None)` that style is cleaner.